tests/test_api_v2.py

```
@@ -761,3 +762,111 @@ def test_user_lookup_handler_missing(ctx):
+    assert 1 == len(calls)
+
+
+@with_script_context
+def test_cluster_requests_touch_no_bucket_at_all(ctx):
+    # type: (ScriptContext) -> None
+    """Not just the IP bucket. The app bucket is keyed by the application being
+    administered, so charging an admin request to it would refuse a request to
+    disable a busy application because that application is busy; and limit()
+    increments before it checks, so merely evaluating the global bucket would
+    spend a paying application's budget on our own bookkeeping."""
+    ctx.config.cluster.secret = "s3cr3t"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "secret"

(S105)